<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-150</x>
      <y>0</y>
      <width>1231</width>
      <height>711</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #be7fbe;
font: 12pt &quot;Cambria&quot;;
QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid rgb(255, 0, 0);
border-top: 3px solid rgb(255, 0, 0);
border-left:  3px solid rgb(255, 0, 0);
border-bottom: 3px solid rgb(255, 0, 0);

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb(255, 0, 0);
color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="title">
     <string>The APP</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(133, 214, 255);</string>
     </property>
     <property name="title">
      <string>PDF</string>
     </property>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>27</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Create PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>77</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>480</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(133, 214, 255);</string>
     </property>
     <property name="title">
      <string>Stats</string>
     </property>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>27</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Show stats</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>370</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(133, 214, 255);</string>
     </property>
     <property name="title">
      <string>Search</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Write here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>1071</width>
       <height>691</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid #ffc778 ;
border-top: 3px solid #ffc778 ;
border-left:  3px solid #ffc778 ;
border-bottom: 3px solid #ffc778 ;

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb#ffc778 ;
color:rgb(255, 255, 255);
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #be7fbe;
font: 12pt &quot;Cambria&quot;;
QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid rgb(255, 0, 0);
border-top: 3px solid rgb(255, 0, 0);
border-left:  3px solid rgb(255, 0, 0);
border-bottom: 3px solid rgb(255, 0, 0);

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb(255, 0, 0);
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Ajouter projet</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Objet</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Budget</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_Objet">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_Desc">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>110</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_Budget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #ffc778 ;
font: 14pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_ID">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>440</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #ffc778 ;
font: 14pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_find">
        <property name="geometry">
         <rect>
          <x>92</x>
          <y>400</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 223, 223);</string>
        </property>
        <property name="text">
         <string>Mettez ici l'ID pour modifier</string>
        </property>
        <property name="placeholderText">
         <string extracomment="type here the id that you want to modify"/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID </string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>110</y>
         <width>701</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #be7fbe;
font: 12pt &quot;Cambria&quot;;
QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid rgb(255, 0, 0);
border-top: 3px solid rgb(255, 0, 0);
border-left:  3px solid rgb(255, 0, 0);
border-bottom: 3px solid rgb(255, 0, 0);

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb(255, 0, 0);
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Afficher projets</string>
       </property>
       <widget class="QTableView" name="tab_etudiant">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>351</width>
          <height>241</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>30</y>
          <width>311</width>
          <height>241</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>420</y>
         <width>431</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #be7fbe;
font: 12pt &quot;Cambria&quot;;
QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid rgb(255, 0, 0);
border-top: 3px solid rgb(255, 0, 0);
border-left:  3px solid rgb(255, 0, 0);
border-bottom: 3px solid rgb(255, 0, 0);

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb(255, 0, 0);
color:rgb(255, 255, 255);
}</string>
       </property>
       <property name="title">
        <string>Supprimer projet</string>
       </property>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>150</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #ffc778 ;
font: 14pt &quot;Cambria&quot;;</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_IDS">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 223, 223);</string>
        </property>
        <property name="text">
         <string>Mettez ici l'ID pour supprimer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="triBudgetCroissant">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>80</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>461</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #d1d1d1;
    border-radius: 10px;
    padding: 8px;
  
}




</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>                                   Chercher Projet</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonPDF">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>430</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Créé un PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stat">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>80</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>statistique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rechercherParObjet">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>191</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>recherche</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="calendar">
      <attribute name="title">
       <string>Calendar</string>
      </attribute>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>113</x>
         <y>100</y>
         <width>581</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>100</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Calendar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Modify</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Page</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #be7fbe;
font: 12pt &quot;Cambria&quot;;
QTabWidget::pane { /* The tab widget frame */
border-right: 3px solid rgb(255, 0, 0);
border-top: 3px solid rgb(255, 0, 0);
border-left:  3px solid rgb(255, 0, 0);
border-bottom: 3px solid rgb(255, 0, 0);

}

QTabBar::tab:selected, QTabBar::tab:hover {
background:rgb(255, 0, 0);
color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>93</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionlogo"/>
   <addaction name="actionIcon1"/>
   <addaction name="actionIcon2"/>
   <addaction name="actionicon3"/>
   <addaction name="actionicon4"/>
   <addaction name="actionicon5"/>
  </widget>
  <action name="actionIcon1">
   <property name="icon">
    <iconset>
     <normaloff>:/image/xxxxxxxxx-01.png</normaloff>:/image/xxxxxxxxx-01.png</iconset>
   </property>
   <property name="text">
    <string>Icon1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionIcon2">
   <property name="icon">
    <iconset>
     <normaloff>:/image/Untitled-1-01.png</normaloff>:/image/Untitled-1-01.png</iconset>
   </property>
   <property name="text">
    <string>Icon2</string>
   </property>
  </action>
  <action name="actionicon3">
   <property name="icon">
    <iconset>
     <normaloff>:/image/sdf1-01.png</normaloff>:/image/sdf1-01.png</iconset>
   </property>
   <property name="text">
    <string>icon3</string>
   </property>
  </action>
  <action name="actionicon4">
   <property name="icon">
    <iconset>
     <normaloff>:/image/xxxxxxccccccccccccc-01.png</normaloff>:/image/xxxxxxccccccccccccc-01.png</iconset>
   </property>
   <property name="text">
    <string>icon4</string>
   </property>
  </action>
  <action name="actionicon5">
   <property name="icon">
    <iconset>
     <normaloff>:/image/qsdfg-01.png</normaloff>:/image/qsdfg-01.png</iconset>
   </property>
   <property name="text">
    <string>icon5</string>
   </property>
  </action>
  <action name="actionlogo">
   <property name="icon">
    <iconset>
     <normaloff>:/image/logo application.png</normaloff>:/image/logo application.png</iconset>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
